src/Consolonia.Gallery/View/MainWindow.axaml

```diff
@@ -7,6 +7,7 @@
         d:DesignWidth="800"
         d:DesignHeight="450"
         x:Class="Consolonia.Gallery.MainWindow"
+        RequestedThemeVariant="Dark"
         Title="Consolonia Control Gallery">
```

Comment:
Dialog should not bring shade on the background in TurboVision. Investigate this, may be it was done already.